src/prism-live.js

```diff
@@ -365,6 +365,11 @@ var _ = Prism.Live = class PrismLive {
 				this.wrapper.style[prop] = cs[prop];
 				this.textarea.style[prop] = this.pre.style[prop] = "inherit";
 			}
+			console.log(prop)
+		}
+    
+		if (this.textarea.classList.contains('line-numbers')) {
+		    this.textarea.style['padding-left']=cs['padding-left'];
```

Comment:
Why check for `line-numbers`? Does this cause trouble in the general case?

Reply:
Since this adds an inline-style I was trying to avoid overwriting any styles placed on the textarea by end-user's.

Reply:
Let me know if you think this should just be added to the textarea regardless, I can see either.

Reply:
You can check whether the textarea was the primary element (i.e. the one that existed before init) by comparing with `this.source`

Reply:
I tested this fix against the pre>code initialization as well, in both cases the textarea requires the padding.